helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -123,11 +123,11 @@ splunkObservability:
   tracesEnabled: true
   logsEnabled: false
 
-  # Option to send Kubernetes events to Splunk Observability Infrastructure Monitoring as data events:
+  # Option to send Kubernetes objects to Splunk Observability Infrastructure Monitoring as data events:
   # https://docs.splunk.com/Observability/alerts-detectors-notifications/view-data-events.html
-  # To send Kubernetes events to Splunk Observability Log Observer, set both clusterReceiver.eventsEnabled and
+  # To send Kubernetes events to Splunk Observability Log Observer, set both clusterReceiver.objectsEnabled and
   # splunkObservability.logsEnabled to true.
-  infrastructureMonitoringEventsEnabled: false
+  infrastructureMonitoringObjectsEnabled: false
```

Comment:
Please don't rename this config option, `infrastructureMonitoringEvents` is the name of some functionality in Splunk Observability IM, not `infrastructureMonitoringObjects`.